packages/neuron-wallet/src/database/chain/entities/tx-lock.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseEntity, Entity, PrimaryColumn } from 'typeorm'
+
+@Entity()
+export default class TxLock extends BaseEntity {
```

Comment:
`Lock` is mentioned in the name but `lock hash` is its field, which is confusing.

Reply:
I have no good idea for the table name, do you have any suggestions?
I thought if the table name includes the field name, it will change to `txhash_lockhash`, but it looks strange. So I simply name it `tx_lock`.

Reply:
Maybe `tx_lock` is good enough if `tx hash` and `lock hash` are treated as `id`